x-pack/.gitignore

```diff
@@ -4,6 +4,7 @@
 /test/functional/failure_debug
 /test/functional/screenshots
 /test/functional/apps/reporting/reports/session
+/test/reporting/configs/failure_debug/
```

Comment:
@tsullivan i've added this entry related to reporting in the `.gitignore`

Reply:
Thanks! This PR looks good overall from my side.